test/prototype/test_autoround.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+from torchao.prototype.autoround.utils import is_auto_round_available
+
+if not is_auto_round_available():
+    pytest.skip("AutoRound is not available", allow_module_level=True)
+
+import torch
+from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
+    parametrize,
+    run_tests,
+    TestCase,
+)
+from torchao import quantize_
+
+from torchao.dtypes import AffineQuantizedTensor
+from torchao.prototype.autoround.core import (
+    apply_auto_round,
+    prepare_model_for_applying_auto_round_,
+)
+from torchao.prototype.autoround.multi_tensor import MultiTensor
+from torchao.utils import TORCH_VERSION_AT_LEAST_2_5
+
+_AVAILABLE_DEVICES = ["cpu"] + (["cuda"] if torch.cuda.is_available() else [])
+
+
+# Copied from https://github.com/pytorch/ao/pull/721
```

Comment:
btw I simplified the example to just use a single linear

Reply:
Got it, I prefer to use `TwoLinear` to demonstrate that this flow can be applied not only to built-in modules but also to customized modules. Please let me know if that makes sense.

Reply:
yeah sure, sgtm